src/Tools/dotnet-trace/dotnet-trace.csproj

```diff
@@ -14,8 +14,7 @@
   <ItemGroup>
     <PackageReference Include="System.CommandLine" Version="$(SystemCommandLineVersion)" />
     <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="$(MicrosoftDiagnosticsTracingTraceEventVersion)" GeneratePathProperty="true" />
-    <PackageReference Include="Microsoft.OneCollect.RecordTrace" Version="0.1.32221" PrivateAssets="All" GeneratePathProperty="true"
-                      Condition="'$(TargetRid)' != '' and $([System.String]::Copy('$(TargetRid)').StartsWith('linux'))" />
```

Comment:
This is why it wasn't working before. We pack on windows and for global tools things are rid agnostic.

Reply:
I currently don't know enough to alter the packaging strategy safely. Is the current change (always having a PackageReference and filtering for None based on local build/BundleTools/Packing) not preferred?

Just curious what specifically smells, since I'm not yet well versed in build/packaging best practices, and the change was just what I was able to get working.

Reply:
Discussed offline, should be an unconditional package reference, which should be the default behavior